scripts.py

```diff
@@ -779,3 +784,50 @@ def __init__(self):
             SearchIndexMonitor,
             index_name=parsed.works_index,
         )
+
+class LoanReaperScript(Script):
+    """Remove expired loans and holds whose owners have not yet synced
+    with the loan providers.
+
+    This stops the library from keeping a record of the final loans and
+    holds of a patron who stopped using the circulation manager.
+
+    If a loan or (more likely) hold is removed incorrectly, it will be
+    restored the next time the patron syncs their loans feed.
```

Comment:
The patron's holds are kept with whoever owns the DRM server. Overdrive, Bibliotheca, etc. The circulation manager tries to get a picture of a patron's holds through the distributor's API, but this picture is always incomplete because we don't know for sure when a book will be available. The SimplyE client learns about holds from the circulation manager. 

The SimplyE client has the ability to _create_ holds, but it does this by asking the circulation manager to do the work, and the circulation manager asks the distributor to do the work. The distributor is ultimately the one in charge of the holds.